### DISCUSSION TITLE
Dismiss multiple entries in News tab

### DISCUSSION THREAD

**Discussion by H-Sachse** [InfinityLoop1308/PipePipe#265] [Q&A]
I just started using Pipepipe and have an issue with the News tab.
I added multiple Youtube channels and now there are many videos in the News tab.
I already watched most of those so I'd like to dismiss those from the news tab.
I'd like to know how I can dismiss multiple or even all News entries at once.

**Comment by InfinityLoop1308** [comment]
The eye icon in the upper right corner can toggle between showing all videos or unwatched videos.

**Reply by H-Sachse** [reply]
This is not what this request is about.
I know that I can hide watched videos like that.
The issue I have is that I have loads of "unwatched" videos that I want to dismiss from the news feed at once.
Please reopen this.

**Reply by InfinityLoop1308** [reply]
We will not support hiding arbitrary videos in the Feed. It is incompatible with the design.
